src/main.ts

```diff
@@ -81,7 +81,8 @@ async function start () {
             appDkBlue: '#38598a', // same as $app-dk-blue
             error: '#d3272c', // same as $app-red
             success: '#1a9031', // same as $app-green
-            gray7: '#495057' // same as $gray7
+            gray7: '#495057', // same as $gray7
+            gray9: '#212529' // Same as $gray9
```

Comment:
You _may_ want to add these to the Vuetify theme as well, so you can use these named colours in Vuetify components (eg, `color="gray9"`).

Reply:
By doing this, you can already use them within the Vuetify component templates

Reply:
Whoops, I thought this was the styles.scss file. You're all good (and ahead of me).